kitipy/executor.py

```diff
@@ -449,6 +449,17 @@ def _remote(
         returncode = sin.channel.recv_exit_status()
         return subprocess.CompletedProcess(cmd, returncode, stdout, stderr)
 
+    def _wrap_cmd(self, cmd: str) -> str:
```

Comment:
`_wrap_cmd` => `_set_bash_options` ?

Reply:
Rather `_set_shell_options`, to be more generic.